source/guides/stacks/dbrs.rst

```diff
@@ -104,7 +104,7 @@ Firmware configuration
 
    When updating DCPMM Firmware, all DCPMM parts must be in the same mode (you cannot mix 1LM and 2LM parts).
 
-The latest firmware download for the Intel® Server System S2600WF Family is available at the `Intel Download Center`_
+The latest firmware download for the Intel Server System S2600WF is available at the `Intel Download Center`_
```

Comment:
Revert to original: Intel® Server System, per DB.
Yes on removing "Family"

Reply:
I think we can leave the (r) off as this is second use of name on the page. The specific system (s2600wf) isn't part of the name.